docs/app.md

```diff
@@ -4,29 +4,34 @@ An app installed in an organization from a repository.
-| `sourceUrl`       | `String`                           | URL of the app source code.                                                                                              |
-| `version`         | `String`                           | The current version of the app.                                                                                          |
-| `kernelAddress`   | `String`                           | The address of the kernel.                                                                                               |
-| `isForwarder`     | `Boolean`                          | Whether the app can act as a forwarder.                                                                                  |
-| `tags`            | `String[]`                         | Tags associated with the app.                                                                                            |
+| Name                | Type                               | Description                                                                                                              |
+| ------------------- | ---------------------------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| `abi`               | `Abi`                              | The Ethereum contract ABI of the app contract.                                                                           |
+| `address`           | `String`                           | The address of the app proxy contract (never changes).                                                                   |
+| `appId`             | `String`                           | The appName but encoded.                                                                                                 |
+| `appName`           | `String`                           | The app ENS identifier. E.g. `"token-manager.aragonpm.eth"`                                                              |
+| `author`            | `String`                           | App author, from the repository. E.g. `"Aragon Association"`.                                                            |
+| `chainId`           | `String`                           | Chain ID for this app.                                                                                                   |
+| `codeAddress`       | `String`                           | The address of the app contract (changes with every major version).                                                      |
+| `contentUri`        | `String`                           | The location of the app content. Empty for special apps like the kernel. E.g. `"ipfs:QmdLEDDfi…"`                        |
+| `contentUrl`        | `String`                           | The HTTP URL of the app content. Uses the IPFS HTTP provider. E.g. `http://gateway.ipfs.io/ipfs/QmdLEDDfi…/`             |
```

Comment:
I'm not sure exactly how to get the `contentUrl`, hatmlUrl` values. Are they the `contentUri` of repos when http is used instead of ipfs?

Reply:
Oh perfect, thanks for the clarification. I agree let's remove `htmlPath`.